Updated Support index.md (#2194)
OctopusDeploy/docs: src/pages/docs/support/index.md

navOrder: 160
---

> If you want to enquire about purchasing, renewing or upgrading your Octopus license, please contact our [sales](https://octopus.com/company/contact) and [customer success](customersuccess@octopus.com) teams. For specific issues about the product please contact our support team.

Reviewer: ```suggestion
To enquire about purchasing, renewing, or upgrading your Octopus license, please [contact our sales team](https://octopus.com/company/contact). 

If you need product help and have a paid license, or you're trialing Octopus, please [email our support team](mailto:support@octopus.com). You can also visit our [Support page](https://octopus.com/support).
```

> For customers with a paid license, or if you're currently trialing Octopus, please reach out for support at [support@octopus.com](support@octopus.com). Otherwise please visit our [Support page](https://octopus.com/support).

Reviewer: ```suggestion
If you need product help and have a paid license, or you're trialing Octopus, please [email our support team](mailto:support@octopus.com). You can also visit our [Support page](https://octopus.com/support).
```

> Sometimes when you contact support we might ask you to perform tasks on your Octopus. This section explains how to perform some of those tasks.

Reviewer: ```suggestion
Sometimes when you contact support, we might ask you to perform tasks in Octopus. This section explains how to perform some of those tasks.
```

:::div{.hint}

> Priority Support is available as an addition to your Octopus Enterprise license. For more information on what is included, please see our [Priority Support page](https://octopus.com/support/priority).

Reviewer: ```suggestion
Priority Support is available as an addition to your Octopus Enterprise license. For more information, please see our [Priority Support page](https://octopus.com/support/priority).
```
